api/api-doc/storage_service.json

```diff
@@ -2396,6 +2404,10 @@
             "threshold":{
                "type":"long",
                "description":"The slow query logging threshold in microseconds. Queries that takes longer, will be logged"
+            },
+            "fast":{
+               "type":"boolean",
+               "description":"Is lightweight tracing mode enabled. In that mode tracing ignore events and tracks only sessions."
```

Comment:
The word "enabled" here and also above can be a bit confusing, because the "enabled" flag is actually separate from the "fast" bit. If I understand correctly, using `fast=true` is not enough - you need to use `fast=true&enable=true`.

Reply:
yes, you are correct. there was a previous implementation version in which "fast" was enabling omitting of the tracing events globally. but that's not the case rn.

Reply:
fixed #8373